README.md

```diff
@@ -0,0 +1,28 @@
+[![Build Status](https://travis-ci.org/RIOT-OS/applications.svg?branch=master)](https://travis-ci.org/RIOT-OS/applications)
+
+# RIOT Applications
+
+This repository provides more applications for the [RIOT operating system][riot-repo].
+Some of them are just useful tools for development work, others showcase
+software provided with RIOT beyond the simple examples within the RIOT codebase.
+
+## Usage
+
+To build and use them follow [the instructions in the RIOT repository][getting-started]
+and the respective application's README.
+
+The submodule to the RIOT repository in this repository will always point to the
```

Comment:
In this application repository, the RIOT main code base is included as submodule, which points to the latest RIOT release.